packages/Statie/src/Console/Application.php

```diff
@@ -2,12 +2,18 @@
 
 namespace Symplify\Statie\Console;
 
+use Jean85\PrettyVersions;
 use Symfony\Component\Console\Application as SymfonyApplication;
 use Symfony\Component\Console\Input\InputDefinition;
 use Symfony\Component\Console\Input\InputOption;
 
 final class Application extends SymfonyApplication
 {
+    public function __construct(string $name = 'Statie', string $version = 'UNKNOWN')
+    {
+        parent::__construct($name, PrettyVersions::getVersion('symplify/statie')->getPrettyVersion());
```

Comment:
I'd either use ctor values or these arguments, not to mix both

Reply:
So what should I put here? Parent default?

Reply:
What do you default constructor for since it's our final class?